src/components/TableToolbar.js

```diff
@@ -312,7 +312,7 @@ class TableToolbar extends React.Component {
               }
             />
           )}
-          {options.filter && (
+          {options.filter && options.filtersInline !== true && (
```

Comment:
I actually would like to see us keep the filters on, even if using inline filters. I can think of use cases for this, but if someone wants to hide the toolbar filter, they already have the filter option here to do so.